sdk/trace/batch_span_processor.go

```diff
@@ -112,7 +112,7 @@ func NewBatchSpanProcessor(exporter export.SpanExporter, options ...BatchSpanPro
 }
 
 // OnStart method does nothing.
-func (bsp *BatchSpanProcessor) OnStart(sd *export.SpanData) {}
+func (bsp *BatchSpanProcessor) OnStart(sd *export.SpanData, parent context.Context) {}
```

Comment:
I would say that the `parent context.Context` parameter should go first, as we have agreed during the SIG meeting that idiomatic go trumps the strict specification adherence. And, idiomatically, context parameter should be the first one.

Reply:
Good point about idiomatic Go > spec. Looks like OTel Java did the same. Thanks for the feedback!